### ISSUE TITLE
Bridge in daemon mode not terminating when shutting down the pc on linux

### ISSUE THREAD

**Issue by luca-demma** [hensm/fx_cast#230]
PopOS 22.04 (Ubuntu based)
Bridge version: v0.2.0

**Description**
When using the bridge in daemon mode if I shutdown the pc the daemon doesn't terminate getting the "A stop job is running for service fx_cast daemon"

This the configuration file of the daemon I'm using:

```
[Unit]
Description=fx_cast daemon

[Service]
User=fx_cast
ExecStart=/opt/fx_cast/fx_cast_bridge -d
Restart=always

[Install]
WantedBy=multi-user.target
```

**Steps to reproduce**
1. Set the bridge in daemon mode on linux as service
2. Shutdown or restart the pc (if the error is not shown press canc while shutting down)

**Expected behaviour**
Shutting down the pc stopping the daemon

**Comment by hensm** [comment]
Ah, I guess the bridge doesn't handle `SIGTERM` properly.
This should fix it for the next version: 0753c70

**Comment by luca-demma** [comment]
Installed the new version, the problem persists

**Comment by hensm** [comment]
Can you test this build?
[fx_cast_bridge-0.3.0-x64.deb.zip](https://github.com/hensm/fx_cast/files/9602836/fx_cast_bridge-0.3.0-x64.deb.zip)

**Comment by luca-demma** [comment]
Tryed right now, same problem

Il mar 20 set 2022, 01:08 Matt Hensman ***@***.***> ha
scritto:

**Comment by hensm** [comment]
Just tested it in a VM, it seems to fix it on my end compared to the released version which _does_ hang. Possibly the package upgrade didn't work for you.

**Comment by luca-demma** [comment]
you're right, I had to uninstall and reinstall to make it work, apparently installing it ontop of the old one didn't work (maybe because they have the same version number)

TL;DR; it works now thanks 😀
